chapters/dae.tex

```diff
@@ -102,10 +102,11 @@ \chapter{Modelica DAE Representation}\label{modelica-dae-representation}
 \end{example}
 
 The generated set of equations is used for simulation and other analysis activities.
-Simulation means that an initial value problem is solved, i.e., initial values have to be provided for the states $x$, \cref{initialization-initial-equation-and-initial-algorithm}.
+In this specification simulation means that first that initial values are found for the states $x$, \cref{initialization-initial-equation-and-initial-algorithm}.
+Given those initial values the equations are simulated forward in time, this is also called solving an initial value problem (in contrast to boundary value problems).
```

Comment:
How about adding _initial value problem_ to the index?  I think we should avoid even mentioning boundary value problems if we are not explaining what they are:
```suggestion
Given those initial values the equations are simulated forward in time, this is also called solving an \firstuse{initial value problem}.
```

If we don't want it in the index, we should use the special form `\firstuse[---]{…}`.